src/main/scala/com/ing/wbaa/airlock/proxy/api/directive/ProxyDirectives.scala

```diff
@@ -71,9 +71,10 @@ object ProxyDirectives extends LazyLogging {
                   .head.split("=")
 
                 val delimiter = queryString
-                  .split("&")
-                  .filter(_.contains("delimiter"))
-                  .head.split("=").last
+                  .split("&").find(_.contains("delimiter")) match {
+                    case Some(d)     => d
+                    case None        => "/"
+                  }
```

Comment:
the parameter is e.g. "delimiter=sth"?
Previously we had `.head.split("=").last` and the result was "sth" and now we have "delimiter=sth" ?
Is this correct?